.travis.yml

```diff
@@ -1,37 +1,38 @@
-language: go
-
 addons:
   hosts:
     - quic.clemente.io
 
-go:
-  - 1.x
-  - tip
-
 matrix:
-  allow_failures:
-    - go: tip
-  fast_finish: true
+  include:
+    - language: go
+      go: 1.11.x
+      env: GO111MODULE=on
+    - language: go
+      go: 1.x
+      env: GO111MODULE=auto
+    - language: go
+      go: tip
+      env: GO111MODULE=auto
```

Comment:
We just need `1.x` and `tip` with `GO111MODULE=on`

Reply:
I'm not so sure.  I believe travis will clone this to an old `GOPATH` style directory in which case it will use the old way of vendor (https://github.com/golang/go/wiki/Modules#when-do-i-get-old-behavior-vs-new-module-based-behavior).